ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -10,11 +11,47 @@ export function generateMetadata(ldkSettings: LdkSettings): string {
   if (!ldkSettings || !ldkSettings.ldk || Object.keys(ldkSettings.ldk).length === 0) {
     throw new Error(permissionsErrorMessage);
   }
+
+  if (ldkSettings.ldk.configSchema) {
+    const ajv = new Ajv();
+    const errors: string[] = [];
+
+    Object.values(ldkSettings.ldk.configSchema.properties as LdkConfigProperties).forEach(
+      (configChild) => {
+        try {
+          ajv.compile(configChild);
+
+          const { default: defaultValue, type } = configChild;
+
+          // Only allow types supported by Loop Library form
```

Comment:
Which loop library form?

Reply:
Heya, Bryan! The form is mentioned [here](https://coda.io/d/Loop-Config_dE8VVdCUd8q/Loop-Config_suO_h#_luGNs) but @alexmatthew-olive would have more detail. It's the form that will be generated from the schema, that org admins will fill out to define the variables.